### ISSUE TITLE
Network configuration wrong for static IP

### ISSUE THREAD

**Issue by jjlauer** [rockstor/rockstor-core#556]
Looks like the NETMASK config value for centos 7 does not work correctly.  If you configure a static ip using the web gui and set a NETMASK of 255.255.255.0, the actual netmask is a /8.  Looks like centos prefers the PREFIX=24 config parameter instead of using a NETMASK config value.

**Comment by schakrava** [comment]
@jjlauer I tried to reproduce this, but couldn't.

I configured an interface with following credentials from the web-ui:
ip address: 192.168.1.133
netmask: 255.255.255.0
gateway: 192.168.1.1

The web-ui shows 192.168.1.133 and netmask = 255.255.255.0, so thats /24.

The output of ifconfig also looks right:
inet 192.168.1.133  netmask 255.255.255.0  broadcast 192.168.1.255

So does the output of nmcli d show:
IP4.ADDRESS[1]:ip = 192.168.1.133/24, gw = 192.168.1.1

Am I missing something?
